cmd/ttn-lw-cli/commands/root.go

```diff
@@ -113,11 +113,21 @@ func preRun(tasks ...func() error) func(cmd *cobra.Command, args []string) error
 		if config.Insecure {
 			api.SetInsecure(true)
 		}
-		if config.CA != "" {
-			pemBytes, err := ioutil.ReadFile(config.CA)
+
+		useCA := true
+		var pemBytes []byte
+		if config.CAFile != "" {
+			pemBytes, err = ioutil.ReadFile(config.CAFile)
 			if err != nil {
 				return err
 			}
+		} else if config.CA != "" {
```

Comment:
IMO `CA` should take precedence. Ideally the flag specified on the command-line should take precedence, in fact, but not sure how to achieve that with cobra @htdvisser ?

Reply:
Coming back to it, I think we should not really allow using both. It might be better to just exit with a useful error message.